packages/esm-patient-appointments-app/src/appointments/appointments.helpers.ts

```diff
@@ -0,0 +1,15 @@
+export type amPm = 'AM' | 'PM';
+
+export const convertTime12to24 = (time12h, timeFormat: amPm) => {
```

Comment:
If I remember correctly we already have a similar function somewhere else (I think it had something to do with visits). If so, I'd move it to the common-lib to avoid duplication.